python/hail/docs/tutorials/introduction-to-the-expression-language.ipynb

```diff
@@ -0,0 +1,1465 @@
+    "\n",
+    "First, a short demonstration of some of the methods accessible on `Variant` and `Genotype` objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v.contig')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v.start')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v.ref')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v.altAlleles')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v.altAlleles.map(aa => aa.isSNP())')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('v.altAlleles.map(aa => aa.isInsertion())')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('g')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('g.dp')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('g.ad')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('g.fractionReadsRef()')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "hc.eval_expr_typed('g.isHet()')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Wrangling complex nested types\n",
+    "\n",
+    "Structs and Arrays allow arbitrarily deep grouping and nesting of values."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The type of `sa` is:\n",
```

Comment:
I can't reproduce this. In the rendered HTML:
![image](https://user-images.githubusercontent.com/10562794/26884894-c58e22d2-4b55-11e7-92a7-1038a5ea0faf.png)

in the nb:
![image](https://user-images.githubusercontent.com/10562794/26884905-d4398a1a-4b55-11e7-96e8-b3ee078e72d2.png)